providers/ringcentral/src/lib/ringcentral.provider.ts

```diff
@@ -0,0 +1,67 @@
+import {
+  ChannelTypeEnum,
+  ISendMessageSuccessResponse,
+  ISmsOptions,
+  ISmsProvider,
+} from '@novu/stateless';
+
+const RC = require('@ringcentral/sdk').SDK;
```

Comment:
Can we use the import syntax here? 
```suggestion
import { SDK as RC } from '@ringcentral/sdk;
```